src/ElmFormat/World.hs

```diff
@@ -44,9 +47,16 @@ instance World IO where
     writeFile = Prelude.writeFile
     writeUtf8File path content = ByteString.writeFile path $ Text.encodeUtf8 content
 
-    doesFileExist = Dir.doesFileExist
-    doesDirectoryExist = Dir.doesDirectoryExist
+    doesFileExist path = do
+        exists <- try (Dir.doesFileExist path) :: IO (Either IOException Bool)
+        return $ Either.fromRight False exists
+
+    doesDirectoryExist path = do
+        exists <- try (Dir.doesDirectoryExist path) :: IO (Either IOException Bool)
+        return $ Either.fromRight False exists
+
     listDirectory = Dir.listDirectory
+    makeAbsolute = Dir.makeAbsolute
```

Comment:
This might also fail but I'm not sure what to do or if we want to handle this :thinking: